upcloud/request/load_balancer.go

```diff
@@ -56,11 +56,11 @@ func (r *DeleteLoadBalancerRequest) RequestURL() string {
 }
 
 type GetLoadBalancerBackendsRequest struct {
-	UUID string `json:"-"`
+	ServiceUUID string `json:"-"`
```

Comment:
Even though there is no other `UUID` to pass here, having just `UUID` here was a bit inconsistent with the rest of the request types for load balancer sub-entities (they all use `ServiceUUID`).

Reply:
Yep that makes sense. Although I would consider naming it `LoadBalancerUUID ` ? Our API documents do talk about "service" and "services" but in SDK it's still basically `LoadBalancer` type's ID.